client_test/cli_test.py

```diff
@@ -361,6 +361,36 @@ def test_volume_get(set_env_client):
             assert f.read() == "foo bar baz"
 
 
-def test_environment_flag(set_env_client, test_dir, servicer):
-    stub_file = test_dir / "supports" / "app_run_tests" / "default_stub.py"
-    _run(["run", "--env=staging", str(stub_file)])
+@pytest.mark.parametrize("command", [["run"], ["deploy"], ["serve", "--timeout=1"], ["shell"]])
+@pytest.mark.usefixtures("set_env_client", "mock_shell_pty")
```

Comment:
Just found this: `pytest.mark.usefixtures` allows usage of fixtures without injection, for a bit cleaner test function signatures without unused variables...